src/Api/inputs/ApiInput.tsx

```diff
@@ -27,6 +27,16 @@ export function ApiInput({
   onDeleteArrayItem?: () => void;
   parentInputs?: string[];
 }) {
+  const onInputChange = (value: any) => {
+    onChangeParam(parentInputs, param.name, value);
+  };
+
+  useEffect(() => {
+    if (param.default) {
+      onInputChange(param.default);
```

Comment:
This will reset the value every time ApiPlayground changes tabs. The logic needs to move to `mint`.

Reply:
Closing PR